Disconnected installation instructions on RHEL 8.6 (#1398)
theforeman/foreman-documentation: guides/common/modules/proc_configuring-the-base-operating-system-with-offline-repositories-in-rhel-8.adoc

[id="configuring-the-base-operating-system-with-offline-repositories-in-rhel-8_{context}"]
= Configuring the Base Operating System with Offline Repositories in RHEL 8

Use this procedure to configure offline repositories for the {RHEL} 8 and {ProjectName} ISO images.

.Procedure

> . Create a directory to serve as the mount point for the ISO file corresponding to the base operating system's version.

Reviewer: ```suggestion
. Create a directory to serve as the mount point for the ISO file corresponding to the version of the base operating system.
```

+
[options="nowrap"]
----

> # mkdir /media/rhel8-server

Reviewer 2: there is no "rhel8 server", only "rhel8", so that's also what I would use in all directory names etc.

----

. Mount the ISO image for {RHEL} to the mount point.
+
[options="nowrap" subs="+quotes"]
----

> # mount -o loop _rhel8-DVD_.iso /media/rhel8-

Reviewer 2: ```suggestion
# mount -o loop _rhel8-DVD_.iso /media/rhel8
```

> # mount -o loop _rhel8-Server-DVD_.iso /media/rhel8-server

Reviewer 2: ```suggestion
# mount -o loop _rhel8-DVD_.iso /media/rhel8
```

----
+
. To copy the ISO file's repository data file and change permissions, enter:
+
[options="nowrap"]
----
# cp /media/rhel8/media.repo /etc/yum.repos.d/rhel8.repo

> # chmod u+w /etc/yum.repos.d/rhel8-server.repo

Reviewer 2: ```suggestion
# chmod u+w /etc/yum.repos.d/rhel8.repo
```

----

. Edit the repository data file and add the `baseurl` directive.
+
[options="nowrap"]
----
[RHEL8-BaseOS]

> name=Red Hat Enterprise Linux BaseOS 8.6.0

Reviewer 2: ```suggestion
name=Red Hat Enterprise Linux BaseOS
```

the same procedure will be used on 8.7, 8.8 and whatever comes then, so I would not mention the exact RHEL version here.

mediaid=None
metadata_expire=-1
gpgcheck=0

> cost=500

Reviewer: Where does this value come from?

Author: This value was given by the QA.

Reviewer 2: I think it's just the default one in the `media.repo` file on the ISO

> baseurl=file:///media/rhel8-server/BaseOS/

Reviewer 2: ```suggestion
baseurl=file:///media/rhel8/BaseOS/
```

[RHEL8-AppStream]

> name=Red Hat Enterprise Linux Appstream 8.6.0

Reviewer 2: ```suggestion
name=Red Hat Enterprise Linux Appstream
```

mediaid=None
metadata_expire=-1
gpgcheck=0
cost=500

> baseurl=file:///media/rhel8-server/AppStream/

Reviewer 2: ```suggestion
baseurl=file:///media/rhel8/AppStream/
```
